src/AMP.php

```diff
@@ -174,6 +174,17 @@ public function loadHtml($html, $options = [])
             $options['use_html5_parser'] = true;
         }
 
+        // By default the convertion of img into amp-anim is disabled (because of ressource cost)
```

Comment:
Alright .. I created a new issue with a checklist of what needs to be done here: #168